static/app/components/nav/utils.tsx

```diff
@@ -102,12 +104,14 @@ export function makeLinkPropsFromTo(to: string): {
   state: object;
   to: LocationDescriptor;
 } {
-  const [pathname, search] = to.split('?');
+  const [rawPathname, search] = to.split('?');
+  const [pathname, hash] = rawPathname.split('#');
```

Comment:
Why are we doing this and not using [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL)

Reply:
Originally because these were path strings and not fully formed URL strings, but great point! I switched this over in 25a82c04624